apps/api/src/app/groups/groups.controller.ts

```diff
@@ -41,8 +41,11 @@ export class GroupsController {
     @ApiQuery({ name: "adminId", required: false, type: String })
```

Comment:
```suggestion
    @ApiQuery({ name: "adminId", required: false, type: String })
    @ApiQuery({ name: "memberId", required: false, type: String })
```

It would be nice to add this line so that the API Docs knows that it's not a required parameter.